### PR TITLE
Fix failed e2e test for --pass-config

### PR DESCRIPTION
**Pull request by asvetlov**

### CODE REVIEW

File: tests/e2e/assets/neuromation-client/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-slim
+FROM python:3.7-slim
```
Comment (serhiy-storchaka) [inline tests/e2e/assets/neuromation-client/Dockerfile]: Is it required? 3.6 is the minimal requirement for the client.

File: tests/e2e/test_e2e_jobs.py [hunk 2]
```diff
@@ -669,7 +670,10 @@ def test_job_run(helper: Helper) -> None:
 ) -> AsyncIterator[str]:
     image = await generate_image(docker)
     yield image
-    await docker.images.delete(image, force=True)
+    try:
```
Comment (serhiy-storchaka) [inline tests/e2e/test_e2e_jobs.py]: While you are here, do you mind to add `try`/`finally` around `yield image`? I was going to do this in a separate PR.

File: tests/e2e/assets/neuromation-client/Dockerfile (hunk 1 above)
Comment (asvetlov) [reply tests/e2e/assets/neuromation-client/Dockerfile]: Technically 3.6 works as well. For 3.7 the installation is slightly faster because a couple of packages are skipped.

File: tests/e2e/test_e2e_jobs.py (hunk 2 above)
Comment (asvetlov) [reply tests/e2e/test_e2e_jobs.py]: I've double-checked recently: `pytest` executes fixutre's tear-down part even if the test raises an exception (doesn't matter is it `AssertionError` or another exception type).
No need to wrap `yield` into `try/finally` block, fixture executor behaves slightly different than `contextlib.[async]contextmanager`.
